ipynb/profiling/kernel_functions_profiling.ipynb

```diff
@@ -119,13 +119,43 @@
     "            \"cpufreq\"        : {\n",
     "                \"governor\" : \"performance\",\n",
     "            },\n",
+    "            # provide a set of files and values to write into them\n",
```

Comment:
The idea of the example notebooks (btw this should be moved under examples/profiling), it's to show the bear minimum required to use a certain API.

This notebook's goal is to show how to profile kernel functions and for that in general it's not required to write files.
Thus, I would probably prefer to keep this notebook as much generic as possible.

Maybe we can add a simple example of files write in this other notebook:
   https://github.com/ARM-software/lisa/blob/master/ipynb/utils/testenv_example.ipynb
which is devoted to show how the test env can be configured.
Moreover, this notebook as well has to be generic... thus perhaps we can just add a couple of examples:
```python
"files"          : {
     # If filenames start with '/' then we do not check if the content
     # matches what we wrote, or even if the write was allowed
     # (i,e, the file existed, we have write permission, etc.)
     # this means we can use this function for sched_features as well
     # as for sysctls whose presence or permissions might depend
     # upon kernel config or OS support.
     # If filenames start with '!/' then we do check that the file exists
     # and the configured value has been properly written.
     '/this_file_doesnt_exist_and_we_dont_care'     : 1
     '!/proc/sys/kernel/sched_latency_ns'           : 16000000
}
```